engine/src/main/java/org/hibernate/search/query/engine/spi/QueryDescriptor.java

```diff
@@ -11,6 +11,10 @@
 /**
  * An index query which is able to create a corresponding {@link HSQuery} object.
  *
+ * <p><strong>Note:</strong>In the Hibernate-Search-ORM module, you may pass query descriptors to
```

Comment:
The capitalization of the module name is weird. AFAICT, the module is named orm.